src/containers/trends/TrendsSVGContainer.js

```diff
@@ -67,9 +68,27 @@ export class TrendsSVGContainer extends React.Component {
     ]);
   }
 
+  renderNoDataMessage(dataType) {
+    const { containerHeight: height, containerWidth: width, margins } = this.props;
+    const xPos = (width / 2) - margins.left + margins.right;
+    const yPos = (height / 2) - margins.top + margins.bottom;
```

Comment:
Now that this component is being rendered one level higher in the tree, it needs to be shifted to the left and down because now it's *outside* the group where we already had the margins included. So basically just delete the subtraction of the margins here and it should be centered.

This is your code:
<img width="981" alt="screenshot 2016-10-17 17 23 46" src="https://cloud.githubusercontent.com/assets/1588547/19460226/8dc7ace8-948e-11e6-87f6-c02d5705916c.png">

This is prod:
<img width="980" alt="screenshot 2016-10-17 17 23 57" src="https://cloud.githubusercontent.com/assets/1588547/19460230/9666a75a-948e-11e6-9e9e-09ac2be01ec0.png">